### PR TITLE
Setting to control display of code and value in advanced search result

### PR DESCRIPTION
**Pull request by MikaelNordberg**
- Added admin setting to control the display of code and value in the advanced search result for variable values
- Changed text color for the many values panel

### CODE REVIEW

**Comment by cklinton** [comment]
Ok

**Review by cklinton — APPROVED** [review]
ok

**Review by likp — COMMENTED** [review]
You should also update the CHANGES.txt file. Otherwise it looks good.

File: PCAxis.Web.Controls/SearchValues/SearchValuesCodebehind.vb [hunk 1]
```diff
@@ -390,12 +390,12 @@ Public Class SearchValuesCodebehind
...
-                Case Else
-                    liS = New ListItem(v.Text, v.Code)
-            End Select
+            If Marker.AlwaysShowCodeAndTextInSearchResult Then
+                liS = New ListItem(v.CodeAndValue, v.Code)
+            Else
+                liS = New ListItem(v.Text, v.Code)
+            End If
+
             liS.Selected = True
```
Comment (JohannesFinsveen) [inline PCAxis.Web.Controls/SearchValues/SearchValuesCodebehind.vb:399]: new ListItem(v.Code, v.Code) is not very usefull, but when ignoring Me.Marker.Variable.PresentationText we change how the system behaves even if the new setting is false. 
Is this on purpose?

**Review by JohannesFinsveen — APPROVED** [review]
Very good.

**Comment by MikaelNordberg** [comment]
> You should also update the CHANGES.txt file. Otherwise it looks good.

Now CHANGES.TXT is updated

File: PCAxis.Web.Controls/SearchValues/SearchValuesCodebehind.vb (hunk 1 above)
Comment (JohannesFinsveen) [reply PCAxis.Web.Controls/SearchValues/SearchValuesCodebehind.vb:399]: Excelent!

**Review by likp — APPROVED** [review]
Looks great now!

**Pull request merged**